src/content/docs/en/pages/secure-journey/advanced-configurations/create-waf-rule-set.mdx

```diff
@@ -0,0 +1,166 @@
+---
+title: How to create a WAF rule set
+description: Create rule sets on WAF 
+meta_tags: edge, secure, waf, rule set, rules engine
+namespace: docs_secure_waf_rule_set
+menu_namespace: secureMenu
+permalink: /documentation/products/guides/secure/create-waf-rule-set/
+---
+
+[Web Application Firewall (WAF)](/en/documentation/products/edge-firewall/web-application-firewall/) allows you to create rule sets regarding possible [threat types](/en/documentation/products/edge-firewall/web-application-firewall/#waf-main-settings). You can set these rule sets on Rules Engine for Edge Firewall to 
+
+This guide will create a rule set to block threats from the **SQL Injection** family to prevent attacks that present the `select` word, for example. After [creating an edge firewall](/en/documentation/products/guides/secure/configure-main-settings/) and [activating the WAF module](/en/documentation/products/guides/secure/configure-main-settings/#modules), follow the steps described next.
```

Comment:
```suggestion
This guide will create a rule set to block threats from the **SQL Injection** family to prevent attacks that attempt a `select` query, for example. After [creating an edge firewall](/en/documentation/products/guides/secure/configure-main-settings/) and [activating the WAF module](/en/documentation/products/guides/secure/configure-main-settings/#modules), follow the steps described next.
```